rasa/core/policies/ensemble.py

```diff
@@ -449,18 +450,24 @@ def _check_if_rule_policy_used_with_rule_like_policies(
             )
 
 
-class Prediction(NamedTuple):
-    """Stores the probabilities and the priority of the prediction."""
-
-    probabilities: List[float]
-    priority: int
-
-
 class SimplePolicyEnsemble(PolicyEnsemble):
+    """Default implementation of a `Policy` ensemble."""
+
     @staticmethod
     def is_not_memo_policy(
-        policy_name: Text, max_confidence: Optional[float] = None
+        policy_name: Optional[Text], max_confidence: Optional[float] = None
```

Comment:
No as this would otherwise evaluate to `True` for the call [here](https://github.com/RasaHQ/rasa/blob/82e98163d3701f490716760213b92619283a4a00/rasa/core/test.py#L575)